Gemfile

```diff
@@ -1,35 +1,54 @@
 source "https://rubygems.org"
 ruby "2.3.3"
 
-gem "rails", "4.2.8"
+gem "rails", "5.0.0"
+
+
+gem "pg"
+gem "sequel", "~> 4.32"
+gem "sequel-rails", "~> 0.9", ">= 0.9.12"
+
+
+gem "puma", "~> 3.4"
+
 
 gem "aws-sdk", "~> 2"
 gem "aws-sdk-rails", ">= 1.0.1"
+
+
 gem "hashie", "~> 3.4"
 gem "multi_json", "~> 1.11"
-gem "pg"
-gem "sequel-rails", "~> 0.9", ">= 0.9.12"
+gem "yajl-ruby", "~> 1.2", require: "yajl"
```

Comment:
I've noticed there is an upgraded version of this gem:

https://github.com/chef/ffi-yajl

Reply:
Well it uses a different system library - so we should stick with the current gem, while we check if that is available on the CF PaaS